projects/vir/src/environments/environment.prod.ts

```diff
@@ -79,5 +79,6 @@ export const environment = {
       sv: '4378',
       en: ''
     },
-  }
+  },
+  geoserver: 'https://geoserver-dev.laji.fi'
 };
```

Comment:
Should this also have `observationMapOptions: {availableOverlayNameBlacklist: []}`?

Reply:
Good catch, thanks! Fixed.